docs/introduction/console-commands-for-application-management-phing-targets.md

```diff
@@ -220,10 +220,24 @@ Great tool when you want to translate your application into another language.
 
 For more information about translations, see [the separate article](/docs/introduction/translations.md).
 
-## Customize Phing properties (paths etc.)
-You can customize any property defined in `build.xml` via configuration file `build/build.local.properties` (use `build/build.local.properties.dist` as a template).
+## Customization of Phing targets and properties
+<!-- TODO: Remove or update the note below -->
+*Note: This section describes customization of Phing targets in version of Shopsys Framework that hasn't been released yet.*
+*In `v7.2.0` and below, all Phing targets are defined in `build.xml` in the project repository only - you can directly modify them or delete them instead of overwriting them.*
+
+You can override and replace any Phing target or property defined in the `shopsys/framework` package by redefining it in your `build.xml` config.
+
+When you override a Phing target, the original is renamed to `shopsys_framework.TARGET_NAME`.
+
+For example, if you override the `clean` target in your `build.xml`, you can still call the original target by `shopsys_framework.clean`.
+This works in direct calls (`php phing shopsys_framework.clean`), in the `depends` attribute of targets and for the `<phingcall target="TARGET_NAME">` task.
+
+## Local customization of Phing properties (paths etc.)
```

Comment:
I like the fact you have added this section, however, I would like to see a more explicit description - when to override a property in a project's `build.xml` and when to use `build.local.properties` - the information is there but it seems to me the paragraph needs rewriting a little, we can discuss it :wink:

Reply:
I definitely want to extend this section with some real world example :+1: 

The section about `build.local.properties` was already present, I just wanted to differentiate it from overwriting targets and properties in `build.xml` of the project.